.changeset/clean-rivers-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Fixes an issue during sign-up flow where a user lands on the continue step, and proceeds successfully through the sign-up process and gets redirected to AP sign-up due to signUp.id being undefined.
```

Comment:
i'm a bit confused by this. I don't see how the `useEffect` solves the issue.

could you post a before/after video of the interactions ?

Reply:
Here is the issue currently present.


https://github.com/user-attachments/assets/3eaf7f04-6728-4f95-8d97-ded8307cd500

Reply:
@panteliselef added before/after to description